src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```diff
@@ -617,6 +620,30 @@ public function thePromotionsShouldHavePriority(PromotionInterface $promotion, i
         Assert::same($this->updatePage->getPriority(), $priority);
     }
 
+    /**
+     * @When /^I want to manage (this promotion)'s coupons$/
```

Comment:
It can be just `@When I want to manage this promotion coupons`, parameters is not (and will not be) used :)